docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM arvindn05:osc-base
```

Comment:
merging with arvind05? Can it be something else?

Reply:
@emanoelxavier It needs to have a maintainer. Discussed with Arvind.